Sources/TensorFlow/Core/Utilities.swift

```diff
@@ -167,3 +167,176 @@ extension UnsafeMutablePointer where Pointee == CTensorHandle? {
         self.init(unwrapped)
     }
 }
+
+//===------------------------------------------------------------------------------------------===//
+// Hashing
+//===------------------------------------------------------------------------------------------===//
+
+internal extension FixedWidthInteger {
+    @usableFromInline
+    init(fromBytes bytes: ArraySlice<UInt8>, startingAt index: Int) {
+        if bytes.isEmpty { self = 0; return }
+        let count = bytes.count
+        self = Self(0)
```

Comment:
Always prefer initializer delegation in initializers.
```suggestion
        self.init(0)
```